### PR TITLE
Ssk/lastn commits

### PR DESCRIPTION
**Pull request by shaswa**
Send commit messages(subject) to Insights

### CODE REVIEW

File: plugins/ci/pkg/ci/ci.go [hunk 1]
```diff
@@ -345,6 +347,16 @@ func getGitInfo(repoName string) (gitInfo, error) {
 		return info, err
 	}
 
+	var commitMessage string
+	commitMessage, err = GetResultsFromCommand("git", "log", "--pretty=format:%s", "-1")
+	if err != nil {
+		logrus.Warn("Unable to get GIT Commit message")
+		return info, err
```
Comment (makoscafee) [inline plugins/ci/pkg/ci/ci.go:354]: The strategy for `getInfo()` is all or nothing right? . either you get all the info or nothing.

**Review by makoscafee — APPROVED** [review]
LGTM. just had one question.

File: plugins/ci/pkg/ci/ci.go (hunk 1 above)
Comment (shaswa) [reply plugins/ci/pkg/ci/ci.go:354]: Yup, it's all or nothing.

**Review by baderbuddy — APPROVED** [review]
LGTM

**Comment by shaswa** [comment]
Thank you, Barnie and Bader!

**Pull request merged**